src/Pages/Channels.razor

```diff
@@ -765,7 +765,8 @@
     private bool checkDisableCloseChannelButton(Channel channel)
     {
         ChannelOperationRequest? lastRequest = channel.ChannelOperationRequests.LastOrDefault();
-        if (lastRequest == null && channel.CreatedByNodeGuard == false) return false;
+        if (channel.CreatedByNodeGuard == false) return true;
+        if (lastRequest == null) return false;
```

Comment:
Changed this. But I'm not sure why we disable the button

Reply:
Me neither, worth checking, good point, we shall be able to close channels created from outside.